src/grt/src/heatMap.cpp

```diff
@@ -203,12 +203,14 @@ bool RoutingCongestionDataSource::populateMap()
     for (uint y_idx = 0; y_idx < gcell_congestion_data.numCols(); ++y_idx) {
       const auto& cong_data = gcell_congestion_data(x_idx, y_idx);
 
-      if (x_idx + 1 >= x_grid_sz || y_idx + 1 >= y_grid_sz) {
```

Comment:
The code used to check greater than or equal but now only checks equal. Is that safe?

Reply:
x_grid_sz and y_grid_sz come from the GCellGrid.
https://github.com/The-OpenROAD-Project/OpenROAD/blob/b989394af4cf28e07b9bea3617ac1ffdd68f3da2/src/grt/src/heatMap.cpp#L186
https://github.com/The-OpenROAD-Project/OpenROAD/blob/b989394af4cf28e07b9bea3617ac1ffdd68f3da2/src/grt/src/heatMap.cpp#L196-L200
The loops are iterating the grid with limits being set from `gcell_congestion_data.numRows()` `gcell_congestion_data.numCols()`, which also comes from the GCellGrid.
https://github.com/The-OpenROAD-Project/OpenROAD/blob/b989394af4cf28e07b9bea3617ac1ffdd68f3da2/src/grt/src/heatMap.cpp#L191
I think in the original code == would be enough, because these loops can't reach out of the map scope, unless for, any reason, the CongestionMap can be larger than the GCellGrid.

Either way, I can change for safety if you wish :)